packages/react-search-ui-views/src/SingleSelectFacet.js

```diff
@@ -40,9 +40,16 @@ const setDefaultStyle = {
 function SingleSelectFacet({ className, label, onChange, options, values }) {
   const selectOptions = options.map(toSelectOption);
   const selectedFilterValue = values[0];
-  const selectedOption = selectOptions.find(o =>
-    deepEqual(o.value, selectedFilterValue)
-  );
+  const selectedOption = selectOptions.find(o => {
+    if (
+      selectedFilterValue &&
+      selectedFilterValue.name &&
+      o.value.name === selectedFilterValue.name
```

Comment:
This is me being a bit nitpicky so feel free to disregard, but I'm personally not a huge fan of single-letter variable names. Something like `option.value.name === selectedFilter.value.name`  would be easier for me to grok personally.

Reply:
Cool, done.